config/Shared/config_default.php

```diff
@@ -826,6 +827,7 @@
     ReadyForMerchantAppOnboardingTransfer::class => 'merchant-app-events',
     MerchantAppOnboardingStatusChangedTransfer::class => 'merchant-app-events',
     AppConfigUpdatedTransfer::class => 'app-events',
+    PaymentUpdatedTransfer::class => 'payment-events',
```

Comment:
please move to the 
`PaymentCreatedTransfer::class => 'payment-events',` 
to have them grouped